source/iopipe/json/serialize.d

```diff
@@ -719,21 +580,21 @@ private void deserializeImpl(T, JT)(ref JT tokenizer, ref T item, ReleasePolicy
     else
     {
         typeof(item.get()) result;
-        deserializeImpl(tokenizer, result, relPol);
+        deserializeItem(policy, tokenizer, result);
         item = result;
     }
 }
 
 // deserialize a class or interface. The type must either provide a static
 // function that returns the deserialized type, or have a zero-arg constructor.
-private void deserializeImpl(T, JT)(ref JT tokenizer, ref T item, ReleasePolicy relPol) if ((is(T == class) || is(T == interface)))
+private void deserializeImplWithPolicy(P, T, JT)(ref P policy, ref JT tokenizer, ref T item) if ((is(T == class) || is(T == interface)))
 {
     // NOTE: checking to see if it's callable doesn't help, because there could
     // be a bug, and in that case, it tries the other branch.
     static if(__traits(hasMember, T, "fromJSON"))
         // && is(typeof(item = T.fromJSON(tokenizer, relPol))))
     {
-        item = T.fromJSON(tokenizer, relPol);
+        item = T.fromJSON(tokenizer, policy);
```

Comment:
Need to handle both relPol and policy options here, as existing code could do either.